src/app/api/v1/slack/rotation/on-duty-post/route.ts

```diff
@@ -27,7 +27,7 @@ export async function POST(req: NextRequest) {
     }
 
     const DbClient = new PostgresClient(organizationName, rotationName);
-    const { rows } = await DbClient.queryUsersForOrganizationAndRotation(organizationName, rotationName);
+    const { rows } = await DbClient.queryUsersForOrganizationAndRotation(organizationName, rotationName, true);
```

Comment:
hmmmm maybe at this point we might need something different here, something like having a json as the parameters instead. But that's over optimizing something that we don't even know if it'll need another optional parameter